src/openshift/openshiftItem.ts

```diff
@@ -54,6 +55,10 @@ export abstract class OpenShiftItem {
         return validator.isURL(value) ? null : message;
     }
 
+    static isAbsolute(message: string, value: string) {
```

Comment:
Create validator function for unix in addition to this one.
``` JavaScript
static isPosixAbsolute(message: string, value: string) {
   return path.posix.isAbsolute(value);
}
```
then use it in storage mount path validation.